lib/rubocop/cop/style/string_concatenation.rb

```diff
@@ -106,7 +106,13 @@ def replacement(parts)
               end
             end
 
-          "\"#{interpolated_parts.join}\""
+          "\"#{handle_empty_quotes(interpolated_parts).join}\""
+        end
+
+        def handle_empty_quotes(parts)
```

Comment:
What do you mean by "empty quotes" here? Is this an empty string that's basically just ""?

Reply:
Renamed to `handle_quotes`. I mean handle when parts contain quote only like `'"'` or `"'"`.